junitparser/junitparser.py

```diff
@@ -647,15 +643,15 @@ def write(self, filepath: str = None, pretty=False):
 class JUnitXml(Element):
     """The JUnitXml root object.
 
-    It may contains a :class:`TestSuites` or a :class:`TestSuite`.
+    It may contain :class:`TestSuites` or a :class:`TestSuite`.
```

Comment:
Actually, there is no TestSuites class (anymore?).

Reply:
I think this note refers to `<TestSuites>` and `<TestSuite>` tags that can be the root element of the XML.